BasiliskII/src/Unix/video_x.cpp

```diff
@@ -1763,13 +1773,17 @@ bool VideoInit(bool classic)
 
 	// Get screen mode from preferences
 	const char *mode_str;
-	if (classic_mode)
-		mode_str = "win/512/342";
-	else
-		mode_str = PrefsFindString("screen");
+	mode_str = PrefsFindString("screen");
 
 	// Determine display type and default dimensions
-	int default_width = 512, default_height = 384;
+	int default_width, default_height;
+	if (classic) {
+		default_width = 512;
+		default_height = 342;
+	}	else {
```

Comment:
Nit: Weird whitespace.

Reply:
![video_x cpp](https://user-images.githubusercontent.com/3475933/87614578-67848980-c6de-11ea-820f-95f7beb4a051.png)

They are tabs. It is consistent with the rest of the file.

Reply:
I got it. It is hidden in my editor.